test/lit.cfg

```diff
@@ -0,0 +1,480 @@
+#
```

Comment:
Reading further, it looks like you have created this initial lit.cfg file by copy/pasting from (old) flang and adapting, which is sensible. It looks like you are basing it off a very old version though. There was a lot of refactoring of lit files done a few years back and the cfg files are a lot shorter. I got as far as the inferFlang call before remembering so there are a load of comments running up to that point, but few after. I suggest taking another look at what a modern clang lit.cfg file does and basing F18's off of that rather than off of old flang's.

Reply:
This file has being removed. We added another PR for regression tests

Reply:
This file now belongs to PR:https://github.com/flang-compiler/f18/pull/861.